vanilla/vanilla
# Break category-settings scss into its components

Add new components:
* Plank component
* Nestable component - works with Nestable.js
* Breadcrumbs component

Also:
* Adds plank and breadcrumbs to the styleguide
* Removes some unnecessary scss extensions in favour of including mixins
* Updates Nestable.js implementation to use js-nestable-* and nestable-* class names
* Updates categories settings page to use new dashboard components
* Fixes drag-and-drop quirks on categories settings page
* Adds toolbar-main class to category settings search box to make it wider
* Adds a badge-outline class

How to test this:
* Check out the new pages in the styleguide
* Check that everything looks/works well in the category settings page:
    * Nesting and ordering
    * Filtering
    * Updating 'Display As' values with both filtered and unfiltered categories
